### DISCUSSION TITLE
Getting EVTVdues to talk to each other

### DISCUSSION THREAD

**Discussion by Mitch-Siegel** [collin80/SavvyCAN#324] [Q&A]
Hi all - I am trying to develop a software solution for a car without actually having the car itself. I have datalogs collected with GVRET and EVTVdue, and would like to use one of my EVTVdue's to play back data. Then, I want to program the second EVTVdue to respond appropriately. Effectively the first due will be emulating all devices in the car and the second will be the one that gets installed in the vehicle. I am having tons of trouble getting the EVTVs to play nice and recognize data sent by the other one. I have not been able to get it working at all, any suggestions would be much appreciated.

**Comment by Mitch-Siegel** [comment]
I'm quite sure some of this is related to windows serial comms oddities. I will update when I get a chance to try on linux and see if the problems persist.

**Comment by collin80** [comment]
The first thing I'd suggest is to make sure you have proper CAN termination between the two boards. The bus should show 60 ohms resistance between H and L with both boards off. That's 120 ohm resistance on both sides. If you have no resistance then the CAN comm will just flat out not work. I believe even the early EVTVDue boards were all terminated. But, you'll want to check. I have used two boards at the same time so it should work but I do use LINUX 95% of the time so I can't say what Windows might do.

**Reply by Mitch-Siegel** [reply]
Just checked, resistance is perfect. Gave things a try on linux and having better luck getting savvycan connect window to see both EVTVs but still not able to see messages from one on the other. Here's a screenshot of how I've got it set up, let me know if anything is wrong. Thanks!
![image](https://user-images.githubusercontent.com/49328485/113374758-d7ee3c00-933b-11eb-9154-2d65415857bb.png)

**Reply by collin80** [reply]
Are you using two different instances of SavvyCAN? I have never attempted to test that use very much at all. Generally if I'm going to have two capture devices attached at the same time I'll connect them both in the same instance of SavvyCAN. You can connect as many capture devices as you want to one copy of SavvyCAN. Well, in theory. I haven't gone over about 3.
